resources/js/components/SingleMedia.vue

```diff
@@ -117,7 +139,8 @@
         if (this.$refs.image.complete) {
           this.width = this.$refs.image.naturalWidth;
           this.height = this.$refs.image.naturalHeight;
-          this.ratio = Math.round((this.width / this.height) * 100) / 100;
+          this.isSocial = this.image.custom_properties.is_for_social_media;
+          this.category = this.categoryOptions[this.image.custom_properties.category] || 'Ukendt';
```

Comment:
Er ret sikker på du kan loade translation således:
```suggestion
          this.category = this.categoryOptions[this.image.custom_properties.category] || this.__('Unknown');
```